vault/dendron.tutorial.navigation-basics.md

```diff
@@ -2,47 +2,55 @@
 id: gHdxXlNMr1w4xqee0n-Mb
 title: Navigation Basics
 desc: 1. Navigation Basics
-updated: 1625573454967
+updated: 1639194721158
 created: 1625563862198
 nav_order: 0
 ---
 
 Let's do a brief overview on how to navigate the Dendron UI.
 
-_Quick note on running commands in VS Code:_
+_Quick note on running commands in VSCode:_
 
->If you're unfamiliar with VS Code, the command palette is used to quickly run commands, including the commands for Dendron. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. Throughout the tutorial, if you see instructions telling you to run a command, remember to bring up the command palette and then type the command name to run it.
+>If you're unfamiliar with VSCode, the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) is used to quickly run commands, including the commands for Dendron. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. Throughout the tutorial, if you see instructions telling you to run a command, remember to bring up the command palette and then type the command name to run it.
 
-![Basic UI](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-layout.png)
+![[dendron://dendron.dendron-site/asset.preview#tutorial-dendron-layout,1:#*]]
 
+> 💡 VSCode lets you re-position any of these windows as you'd like, so feel free to rearrange the windows into the view that works best for you!
 #### 1. Editor Pane
```

Comment:
add a newline btw header and the quote, otherwise the header gets formatted as a quote

Reply:
to be consisten with documentation elsewhere and vscode (https://code.visualstudio.com/docs/getstarted/userinterface#_explorer), can we rename this to "Editor View"?